logic/transaction.js

```diff
@@ -583,6 +583,9 @@ Transaction.prototype.verify = function (trs, sender, requester, checkExists, cb
 	}
 
 	// Check timestamp
+	if (trs.timestamp < -2147483648 || trs.timestamp > 2147483647) {
```

Comment:
Assign as constant

Reply:
Where should this constant be stored?

Reply:
`var MAX_INT_32 = 2147483648` in the line 416 logic/transction.js